src/lib/serviceRoutinesV2/postEntity.cpp

```diff
@@ -100,14 +100,14 @@ std::string postEntity
     }
   }
 
-  // Default value for status code: SccCreated
+  // Default value for status code: SccNoContent
   if ((ciP->httpStatusCode == SccOk) || (ciP->httpStatusCode == SccNone) || (ciP->httpStatusCode == SccCreated))
```

Comment:
The point is the following. As far as I understand, ciP->httpStatusCode is filled with the returned value of mongoUpdateContext. However, mongoUpdateContext is unable to return SccCreate code, as this is not a valid code in NGSIv1 (the ones that mongoBackend is able to return).

I'd do an empirical test: remove the (ciP->httpStatusCode == SccCreated) clause and re-run tests.